src/ducks/billsMatching/Linker/billsToOperation/operationsFilters.js

```diff
@@ -45,7 +31,7 @@ const isUncategorizedOperation = operation => {
 }
 
 const isHealthBill = bill => {
-  return includes(HEALTH_VENDORS, bill.vendor)
+  return bill.type === 'health_costs'
```

Comment:
Are you sure there is no risk to have false negatives here ? I mean that we must be sure that every vendor here correctly sets the bill.type. Shouldn't we do 

```
  return  bill.type === 'health_costs' || includes(HEALTH_VENDORS, bill.vendor)
```

Reply:
@doubleface checked that all health connectors sets the `bill.type` to `health_costs`, so I think it is safe